packages/edit-navigation/src/components/menu-editor/block-editor-area.js

```diff
@@ -59,8 +65,11 @@ export default function BlockEditorPanel( {
 	}, [ rootBlockId ] );
 
 	return (
-		<Panel className="edit-navigation-menu-editor__block-editor-panel">
-			<PanelBody title={ __( 'Navigation menu' ) }>
+		<Card className="edit-navigation-menu-editor__block-editor-panel">
+			<CardHeader className="edit-navigation-menu-editor__block-editor-panel-header">
+				{ __( 'Navigation menu' ) }
+			</CardHeader>
+			<CardBody>
 				<div className="components-panel__header-actions">
 					<Button isPrimary onClick={ saveBlocks }>
```

Comment:
I think it would make sense to put the `<Button>` component up within the `<CardHeader>` component, and maybe position it to the right-side of the header.